includes/class-amp-theme-support.php

```diff
@@ -1808,7 +1800,9 @@ public static function is_output_buffering() {
 	 */
 	public static function finish_output_buffering( $response ) {
 		self::$is_output_buffering = false;
-		return self::prepare_response( $response );
+		$response                  = self::prepare_response( $response );
+		do_action( 'amp_server_timing_send' );
```

Comment:
phpdoc 〃

Reply:
Resolved via https://github.com/ampproject/amp-wp/pull/5005/commits/5dc57b27a159c1f08f36ee2e34f11b47f4c17a71

Reply:
_almost_ 😄  completed in b12922e2b61024378fa5697aa2483b6ac10908a3